client/lcd/lcd_test.go

```diff
@@ -252,6 +252,41 @@ func TestCoinSend(t *testing.T) {
 	assert.Equal(t, int64(1), mycoins.Amount)
 }
 
+func TestIBCTransfer(t *testing.T) {
```

Comment:
isnt this just doing normal send? we dont actually call doIBCTransfer

Reply:
🤦‍♂️ Oops, fixed